ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -96,10 +97,31 @@ describe('DataAccessRequirements', () => {
         expect(activeModule).toEqual(AccessModule.ERACOMMONS)
     });
 
-    // update this if the order changes
-    it('should return the second enabled module (ERA, not RAS) from getActiveModule' +
-        ' when the first module (2FA) has been completed and RAS is disabled', () => {
-        serverConfigStore.set({config: {...defaultServerConfig, enableRasLoginGovLinking: false}});
+    it('should return the second module (ERA) from getActiveModule when the first module (2FA) has been bypassed', () => {
+        const testProfile = {
+            ...profile,
+            accessModules: {
+                modules: [{moduleName: AccessModule.TWOFACTORAUTH, bypassEpochMillis: 1}]
+            }
+        };
+
+        const [navigate, ] = useNavigation();
+        const enabledModules = getEnabledModules(allModules, navigate);
+        const activeModule = getActiveModule(enabledModules, testProfile);
+
+        expect(activeModule).toEqual(allModules[1]);
+        expect(activeModule).toEqual(enabledModules[1]);
+
+        // update this if the order changes
+        expect(activeModule).toEqual(AccessModule.ERACOMMONS)
+    });
+
+    // TODO(RW-7301)
```

Comment:
not related to this bugfix. just an opportunity to clarify what this test intends.